modules/ztp-du-cluster-config-reference.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-vdu-configuration-reference.adoc
+
+:_module-type: REFERENCE
+[id="ztp-du-cluster-config-reference_{context}"]
+= Recommended cluster configurations to run vDU applications
+
+Clusters running virtualized distributed unit (vDU) applications require a highly tuned and optimized configuration. The following information describes the various elements that you require to support vDU workloads in {product-title} {product-version} clusters.
+
+== Recommended cluster MachineConfig CRs
+
+The following `MachineConfig` CRs configure the cluster host:
+
+.Recommended MachineConfig CRs
+[cols=2*, width="80%", options="header"]
+|====
+|CR filename
+|Description
+
+|`02-workload-partitioning.yaml`
+|Configures workload partitioning for the cluster. Apply this `MachineConfig` CR when you install the cluster.
+
+|`MachineConfigSctp.yaml`
+|Loads the SCTP kernel module. This `MachineConfig` CR is optional and can be omitted if you do not require this kernel module.
+
+|`MachineConfigContainerMountNS.yaml`
+|Configures the container mount namespace and kubelet conf.
+
+|`MachineConfigAcceleratedStartup.yaml`
+|Configures accelerated startup for the cluster.
+
+|`06-kdump-master.yaml`, `06-kdump-worker.yaml`
+|Configures kdump for the cluster.
```

Comment:
should kdump be in back ticks? I've seen it both ways. In this context it might not be.